### PR TITLE
projects:ad4696_iio, ad7134_iio: Added github docs

### PR DESCRIPTION
**Pull request by jansunil**
Added github docs for AD4696 and AD7134

### CODE REVIEW

File: doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `AD4695 <https://www.analog.com/en/products/ad4695.html>`_ 
+* `AD4696 <https://www.analog.com/en/products/ad4696.html>`_ 
+
```
Comment (D-Disha) [inline doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:12]: Can we also add the EVAL board links?

File: doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	
+**SDP-K1:**
+
+Connect the VIO_ADJUST jumper on the SDP-K1 board to 3.3V position to drive SDP-K1 GPIOs at 3.3V
+
+**EVAL-AD4696:**
+
+* Make below jumper settings on the board. Refer `EVAL-AD4696-FMCZ User Manual <https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad4696.html#eb-overview>`_  for more details.
+
+   .. image:: /source/projects/ad4696_iio/ad4696_jumper_connection.png
```
Comment (D-Disha) [inline doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:74]: I suppose table should be added as done in ad4130 project

File: doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      :width: 600
+
```
Comment (SaikiranGudla) [inline doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:76]: nit: extra line can be removed

File: doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst (hunk 1 above)
Comment (jansunil) [reply doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:12]: Yes, added them now

File: doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst (hunk 2 above)
Comment (jansunil) [reply doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst]: Added a table format now

**Comment by jansunil** [comment]
changelog:
1) Added a table for jumper connections in AD4696
2) Added links to the EVAL boards supported, from the product page
3) Resolved merge conflicts in the index.rst file

File: doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `AD7134 <https://www.analog.com/en/products/ad7134.html>`_ 
+* `AD4134 <https://www.analog.com/en/products/ad4134.html>`_ 
+
```
Comment (D-Disha) [inline doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst:12]: Can Eval board links be added here as well.

Comment (jansunil) [reply doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst:12]: Relevant boards that can be used with this FW aren't released yet.

Comment (D-Disha) [reply doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst:12]: Can we add a TODO then, like ad719x?

**Review by SaikiranGudla — APPROVED** [review]

File: doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst (hunk 4 above)
Comment (jansunil) [reply doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst:12]: A JIRA has been added to track this - https://jira.analog.com/browse/PCSW-557

**Comment by jansunil** [comment]
Hi All- any review comments on this PR?

**Review by ribdp — APPROVED** [review]

**Pull request merged**